pajbot/bot.py

```diff
@@ -802,6 +802,9 @@ def on_pubnotice(self, chatconn, event):
     def on_clearchat(self, chatconn, event):
         tags = {tag["key"]: tag["value"] if tag["value"] is not None else "" for tag in event.tags}
 
+        if "target-user-id" not in tags:
```

Comment:
Please add a comment:

```suggestion
        # Ignore "Chat has been cleared by a moderator" messages
        if "target-user-id" not in tags:
```